website/developer-docs/docs/templates/index.md

```diff
@@ -8,8 +8,8 @@ The most common types are:
 
 -   [**Procedural**](./procedural.md): these are How To docs, the HOW information, with step-by-step instructions for accomplishing a task. This is what most people are looking for when they open the docs... and best practice is to separate the procedural docs from long, lengthy conceptual or reference docs.
 
--   **Conceptual**: these docs provide the WHY information, and explain when to use a feature (or when not to!), and general concepts behind the feature or functioanlity.
+-   [**Conceptual**](./conceptual.md): these docs provide the WHY information, and explain when to use a feature (or when not to!), and general concepts behind the feature or functionality.
 
--   **Reference**: this is typically tables or lists of reference information, such as configuration values, or most commmonly APIs.
+-   **Reference**: this is typically tables or lists of reference information, such as configuration values, or functions, or most commmonly APIs.
 
```

Comment:
I know you're familiar with the [Divio Quadrants of Documentation].(https://documentation.divio.com/)